src/Util.hpp

```diff
@@ -33,6 +34,23 @@
 
 class Context;
 
+// Helper variadic function to construct native paths.
+// Used like: PATH("usr", "local", "bin");
+template<typename... T>
+std::string
+PATH(T&... args)
+{
+  std::vector<std::string> arg_list{args...};
+  std::filesystem::path full;
+
+  for (auto&& p_str : arg_list) {
+    std::filesystem::path p(p_str);
+    full = full / p;
```

Comment:
Can this be simplified to

```suggestion
    full = full / p_str;
```

?

Reply:
That works, thanks!